includes/polling/ports/os/edgeosolt.inc.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+unset($port_stats);
+
+$sfp_stats = snmpwalk_group($device, 'ubntSfps', 'UBNT-UFIBER-MIB', 1, $sfp_stats);
```

Comment:
```suggestion
$sfp_stats = snmpwalk_group($device, 'ubntSfps', 'UBNT-UFIBER-MIB');
```

Reply:
The detection didn't work before, but I found out that I need to use a different oid.  I reverted foreach and added an offset, without which even if I set unset to port_stats, some ports got the wrong id.